cmd/prometheus/main.go

```diff
@@ -239,6 +239,9 @@ func main() {
 	a.Flag("rules.alert.resend-delay", "Minimum amount of time to wait before resending an alert to Alertmanager.").
 		Default("1m").SetValue(&cfg.resendDelay)
 
+	a.Flag("scrape.timestamp-tolerance", "Tolerance applied to scrapes timestamp to improve timestamp compression. Experimental.").
```

Comment:
Perhaps describe more what it is actually doing than what it is for? e.g.

"Adjust scrape timestamps by up to this amount to align them to the intended schedule."

Reply:
:+1: